webservices/resources/elections.py

```diff
@@ -199,6 +200,7 @@ def _get_aggregates(self, pairs):
             sa.func.sum(sa.func.coalesce(pairs.c.disbursements, 0.0)).label('total_disbursements'),
             sa.func.sum(sa.func.coalesce(pairs.c.cash_on_hand_end_period, 0.0)).label('cash_on_hand_end_period'),
             sa.func.array_agg(sa.distinct(pairs.c.cmte_id)).label('committee_ids'),
+            sa.func.max(pairs.c.coverage_end_date).label('coverage_end_date'),
```

Comment:
I just wanted to flag a quick verification here on the use of `max` being intentional.  I'm less familiar with this data. :-)

Reply:
Yes, we want the most recent coverage end date, that will tell us what this total includes. You won't accidentally compare a fresh report total to a total that has not reported recently.